.github/workflows/mvn-build.yml

```diff
@@ -13,11 +13,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2.3.4
       - name: Set up JDK 11
         uses: joschi/setup-jdk@7134ae95986e4e0a4f9f0b51345c93fcebfc4ea9
         with:
           java-version: openjdk11
       - name: Build
-        run: mvn clean install
+        run: |
+          echo "JAVA_HOME=/opt/hostedtoolcache/AdoptOpenJDK/1.0.0-releases-openjdk11-hotspot-normal-latest/x64" >> $GITHUB_ENV
```

Comment:
I think you should just update `joschi/setup-jdk` to the version we use in Quarkus: `e87a7cec853d2dd7066adf837fe12bf0f3d45e52`

Reply:
At least we didn't have to do any of this on Quarkus.

Reply:
yes, I have taken the Quarkus one, just didn't have the time to push. Thank you!!